plugins/coverage/token.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const boom = require('boom');
+
+module.exports = config => ({
+    method: 'GET',
+    path: '/coverage/token',
+    config: {
+        description: 'Get an access token to talk to coverage server',
+        notes: 'Returns a job record',
+        tags: ['api', 'coverage'],
+        auth: {
+            strategies: ['token'],
+            scope: ['build']
+        },
+        plugins: {
+            'hapi-swagger': {
+                security: [{ token: [] }]
+            }
+        },
+        handler: (request, reply) => {
+            const buildCredentials = request.auth.credentials;
+
+            return config.coveragePlugin.getAccessToken(buildCredentials)
+                .then(reply)
```

Comment:
so we'll just return a string here?

Reply:
yes, I also update the description for the endpoint to make that clear